src/NBitcoin/Block.cs

```diff
@@ -106,6 +106,11 @@ public void ReadWrite(BitcoinStream stream)
         #endregion
 
         public uint256 GetHash()
+        {
+            return GetHash(NetworkOptions.TemporaryOptions);
+        }
+
+        public uint256 GetHash(NetworkOptions options)
```

Comment:
This is a new method so I would rather force options to not be null and not use `TemporaryOptions inside this method.

Reply:
Done